src/projectscene/view/clipsview/clipslistmodel.cpp

```diff
@@ -288,6 +305,25 @@ void ClipsListModel::onTimelineFrameTimeChanged()
     updateItemsMetrics();
 }
 
+void ClipsListModel::setSelectedItems(const QList<ClipListItem *> &items)
+{
+    setItemsSelected(false);
+    m_selectedItems = items;
+    setItemsSelected(true);
+}
+
+void ClipsListModel::addSelectedItem(ClipListItem *item)
+{
+    item->setSelected(true);
+    m_selectedItems.append(item);
+}
+
+void ClipsListModel::clearSelectedItems()
+{
+    setItemsSelected(false);
```

Comment:
I believe that now this method is redundant, it is better to copy its body here explicitly, it will be clearer and this method cannot be used incorrectly (for example, calling it with false, but not clearing the `m_selectedItems` list will result in an incorrect state)